pkg/manager/member/pd_member_manager.go

```diff
@@ -258,18 +261,27 @@ func (pmm *pdMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 
 	pdClient := pmm.pdControl.GetPDClient(tc)
 
-	cluster, err := pdClient.GetCluster()
+	healthInfo, err := pdClient.GetHealth()
```

Comment:
why do you chage the order `pdClient.GetHealth()` and `pdClient.GetCluster()`?

If there is no special need, it is better to stay in the old order.

Reply:
It maybe more intuitive to check pd's health first, in my opinion.  QAQ

Alright, I'll reset the order.